packages/cra-template/template/scripts/coverage-merge.js

```diff
@@ -1,28 +1,34 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
```

Comment:
I'm not sure if we need to remove typescript from our shared ESLint config or not.
Although we moved to use typescript by default, but there are some configurations file that still using JS 🤔

Reply:
@hoangmirs I think this would only happen on a few JS files so keeping the Typescript rule from the shared config and ignoring some of them on a few files still make sense to me 🤔